openlibrary/macros/LoanStatus.html

```diff
@@ -80,11 +80,13 @@
     $ viewbook = "//%s/stream/XXX?ref=ol" % bookreader_host()
     <a href="$viewbook.replace('XXX', page.ocaid)" title="Use BookReader to read online" class="cta-btn cta-btn--available">Read</a>
 
-$elif (not page.get('ocaid') or page.is_access_restricted()) and editions_page:   
+$elif (not page.get('ocaid') or page.is_access_restricted()) and editions_page:
   $ betterworldbooks_metadata = get_betterworldbooks_metadata(isbn)
-  $if user and user.is_sponsor() and isbn and all(page.get(x) for x in ['publishers', 'title', 'publish_date', 'covers']) and page.works and page.works[0].get('authors') and betterworldbooks_metadata and ('price_amt' in betterworldbooks_metadata) and (float(betterworldbooks_metadata['price_amt']) < 15.00):
+  $if user and user.is_sponsor() and isbn and all(page.get(x) for x in ['publishers', 'title', 'publish_date', 'covers']) and
+  page.works and page.works[0].get('authors') and betterworldbooks_metadata and ('price_amt' in betterworldbooks_metadata)
+  and (float(betterworldbooks_metadata['price_amt']) < 15.00):
     $ user_id = user.key.split("/")[-1] if user.key else ''
-    <a href="https://archive.org/donate?context=ol&userid=$(user_id)&campaign=pilot&type=book-sponsorship&isbn=$(isbn)" class="cta-btn cta-btn--sponsor">Sponsor eBook</a>
+    <a href="https://archive.org/donate?context=ol&userid=$(user_id)&campaign=pilot&type=book-sponsorship&isbn=$(isbn)" class="cta-btn cta-btn--sponsor cat-btn--sponsor:link">Sponsor eBook</a>
```

Comment:
I don't think we should need `cat-btn--sponsor:link` here -- also this is typo'd, even if we did need it :)